src/services/announcement/announcementService.ts

```diff
@@ -0,0 +1,46 @@
+import { Repository } from "typeorm";
+import { Announcement } from "../../db/entity/hub";
+import { Cache } from "../../util/cache";
+
+export class AnnouncementService {
+  private announcementRepository: Repository<Announcement>;
+  private cache: Cache;
+  constructor(_announcementRepository: Repository<Announcement>, _cache: Cache) {
+    this.announcementRepository = _announcementRepository;
+    this.cache = _cache;
+  }
+
+  createAnnouncement = async (announcement: Announcement): Promise<void> => {
```

Comment:
Again, the methods don't have access modifiers. We can switch over to not using the modifiers for public methods but whatever choice we make it should be consistent with the entire codebase.
I think that we should use access modifiers because the result of omitting them is different in different languages (e.g.: no access modifier in C# means `internal`) but `public` has the same effect in almost all programming languages so in my opinion it improves the readability of the code.

Reply:
I will add in the access modifiers and make sure they are consistent throughout.